README.md

```diff
@@ -16,15 +22,28 @@
 6. `npm run dev` or `yarn run dev` to run it in development mode.
 7. Go to `http://localhost:{{PORT}}/`   =>{{PORT}} : is the port number that you precise in your `.env` file
 
-### C. Tools used
+## Description 
+
+### Features
```

Comment:
The following are not features, they are the tools we used to develop this app, don't confuse them with features.